JustSaying/AwsTools/MessageHandling/SnsTopicBase.cs

```diff
@@ -111,12 +112,14 @@ public void Publish(Message message)
         private PublishRequest BuildPublishRequest(Message message)
         {
             var messageToSend = _serialisationRegister.Serialise(message, serializeForSnsPublishing: true);
-            var messageType = message.GetType().Name;
+            var messageType = message.GetType().Name;

```

Comment:
Would this create funky names if someone was using a generic type?

Reply:
@martincostello This is a whitespace change, I haven't actually changed anything here.

Reply:
[I was right](https://github.com/justeat/JustSaying/issues/278#issuecomment-395370692) though 😉